modules/ROOT/pages/rest-sdk/simple-composition.adoc

```diff
@@ -0,0 +1,80 @@
+= Simple Composition
+ifndef::env-site,env-github[]
+include::../_attributes.adoc[]
+endif::[]
+
+Simple composition can be used to build a DSL through declarative metadata.
+
+== Simple Composition Patterns
+
+There are specific pre-identified use cases that fit a general pattern in the target API. You can identify these patterns and use declarative metadata to express them.
+
+=== Operation Adapter
+
+This pattern is used when the input or output types of one operation need to be changed.
+
+==== Examples of Operation Adapter
```

Comment:
For each example, I suggest showing before and after. Before what API responds with and after by showing the experience we're creating with our composer connectors.